diff --git a/node_modules/@llamaindex/server/server/app/components/ui/chat/chat-header.tsx b/node_modules/@llamaindex/server/server/app/components/ui/chat/chat-header.tsx
index fa69e29..44a9457 100644
--- a/node_modules/@llamaindex/server/server/app/components/ui/chat/chat-header.tsx
+++ b/node_modules/@llamaindex/server/server/app/components/ui/chat/chat-header.tsx
@@ -40,7 +40,7 @@ function LlamaIndexLinks() {
           alt="Llama Logo"
         />
       </div>
-      <a
+      {/* <a
         href="https://github.com/run-llama/LlamaIndexTS"
         target="_blank"
         rel="noopener noreferrer"
@@ -49,7 +49,7 @@ function LlamaIndexLinks() {
           <Star className="mr-2 size-4" />
           Star on GitHub
         </Button>
-      </a>
+      </a> */}
     </div>
   );
 }
